catalog/from-consul/source.go

```diff
@@ -67,10 +69,40 @@ func (s *Source) Run(ctx context.Context) {
 					break
 				}
 			}
+			if k8s {
+				continue
+			}
 
-			if !k8s {
-				services[s.Prefix+name] = fmt.Sprintf("%s.service.%s", name, s.Domain)
+			// Consul service names have no constraints, they can have any
```

Comment:
I'm not sure if this should be done here or when we're about to create the service in Kubernetes in sink.go.

I put it here because in sink.go, all the maps use the service name as keys so I was worried they'd get out of sync

Reply:
I think sink.go is a more appropriate place to do it since it seems to be more "k8s-aware".